### ISSUE TITLE
Function calls are not supported

### ISSUE THREAD

**Issue by wbhob** [CaliStyle/angular2-intercom#12]
Copied from @craigeddy #11

*When I use this version, I'm getting a compile error on ng serve:*

`ERROR in Error encountered resolving symbol values statically. Calling function 'IntercomModule', function calls are not supported. Consider replacing the function or lambda with a reference to an exported function, resolving symb ol AppModule in C:/Users/eddycr/Source/Repos/registration/src/app/app.module.ts, resolving symbol AppModule in C:/U sers/eddycr/Source/Repos/registration/src/app/app.module.ts`

*the code:*

```ts
import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import { FormsModule } from '@angular/forms';
import { IntercomModule } from 'ng-intercom';
import { AppComponent } from './app.component';
import { RegisterModel } from './app.registermodel';
import { RegistrationComponent } from 'app/registration/registration.component';

@NgModule({
  declarations: [
    AppComponent,
    RegistrationComponent
  ],
  imports: [
    BrowserModule,
    FormsModule,
    IntercomModule.forRoot({ app_id: 'sq739kpn'})
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }

```

*I also have a repo with my whole project if you want to take a look. I'm probably doing/forgetting something noob-ish. [craigeddy/IntercomProtype](https://github.com/craigeddy/IntercomProtype) (angular2-intercom-alpha branch)*

**Comment by wbhob** [comment]
@craigeddy just published v1.0.0-alpha.7, and it seems to be working in my test project. Thanks for all your help testing! Know you're helping all of the other NgIntercom users to get an amazing dev experience!

**Comment by craigeddy** [comment]
Cool. I'll give it a try sometime later this week. Thanks!
